test/addons/rook-pool/start

```diff
@@ -14,18 +14,23 @@ from drenv import kubectl
 
 
 def deploy(cluster):
-    print("Creating RBD pool and storage/snapshot classes")
 
+    print("Creating StorageClass")
     template = drenv.template("storage-class.yaml")
     yaml = template.substitute(cluster=cluster)
-
     kubectl.apply("--filename=-", input=yaml, context=cluster)
+
+    print("Creating RBD pool")
     kubectl.apply(
         "--filename=replica-pool.yaml",
-        "--filename=snapshot-class.yaml",
         context=cluster,
     )
```

Comment:
This can fit in line now, if you send another version, try to remove the trailing "," in the last argument. black will automatically reformat the line if needed or return the "," if the line is too long.

Reply:
done.